Temporary fix to avoid failures related to RHOAIENG-7887

jiripetrlik opened the pull request.

Srihari1192 commented on ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot:
```diff
@@ -37,6 +38,9 @@ Run TestKueueRayGpu ODH test
 
 *** Keywords ***
 Prepare Codeflare E2E Test Suite
+    Log To Console    "Restaring kueue component"
```
```suggestion
    Log To Console    "Restarting kueue component"
```

Srihari1192 commented on ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot:
```diff
@@ -96,3 +100,11 @@ Run Codeflare ODH Test
         FAIL    ${TEST_NAME} failed
     END
+
+Restart Kueue
```
Probably we can move this Method to [DistributedWorkloads.resource](https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource) and can reuse it for DW UI [Ray test](https://github.com/red-hat-data-services/ods-ci/blob/f756474e2f9a449de52568d9f70e1f6f8d3cde49/ods_ci/tests/Tests/650__distributed_workloads/test-distributed-workloads-metrics-ui.robot#L135) workaround  as well

jiripetrlik commented:
@Srihari1192 Good point. I've moved restart of Kueue to DistributedWorkloads.resource. @sutaakar I've also simplified restart procedure to use oc rollout restart command. Updated PR was tested successfully on our Jenkins.
